### PR TITLE
Add option to pass claim file via URL and bugfix

### PR DESCRIPTION
**Pull request by edcdavid**
Adds option to pass feedback and claim file via URL (online website only) and fixed/refactored loading local claimjson.js and feedback.js. The claim file and feedback must be online in the same domain as the main page:
http://localhost/results-embed.html?claimfile=claim.json&feedback=feedback3.json

Fixed several bugs: 
- claimjson.js and feedback.js used the old parsing when loaded. Results failed to load with old claim format 
- Saved result file lost formatting (regression from https://github.com/test-network-function/cnf-certification-test/pull/1145)
- refactor

### CODE REVIEW

File: cnf-certification-test/results/html/results-embed.html [hunk 1]
```diff
@@ -1803,5 +1858,4 @@
   </script>
 </body>
 
-</html>
-
+</html>
```
Comment (sebrandon1) [inline cnf-certification-test/results/html/results-embed.html:1861]: I think this needs the newline.

File: cnf-certification-test/results/html/results.html [hunk 2]
```diff
@@ -1278,5 +1333,4 @@
   </script>
 </body>
 
-</html>
-
+</html>
```
Comment (sebrandon1) [inline cnf-certification-test/results/html/results.html:1336]: Another newline here.

**Review by sebrandon1 — APPROVED** [review]
Some comments about newlines.

File: cnf-certification-test/results/html/results-embed.html (hunk 1 above)
Comment (edcdavid) [reply cnf-certification-test/results/html/results-embed.html]: fixed

**Review by aabughosh — APPROVED** [review]

**Pull request merged**